src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -159,19 +173,36 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
 
   const auto* thread_state_slice =
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
-  uint64_t begin_ns = thread_state_slice->begin_timestamp_ns();
-  uint64_t end_ns = thread_state_slice->end_timestamp_ns();
 
-  return absl::StrFormat(
+  std::string tooltip = absl::StrFormat(
       "<b>%s</b><br/>"
       "<i>Thread state</i><br/>"
       "<br/>"
-      "%s"
+      "%s",
+      GetThreadStateName(thread_state_slice->thread_state()),
+      GetThreadStateDescription(thread_state_slice->thread_state()));
+
+  if (thread_state_slice->wakeup_tid() != orbit_base::kInvalidThreadId) {
+    std::string blocked_or_created = GetWakeupReason(thread_state_slice->wakeup_reason());
+    std::string thread_name = capture_data_->GetThreadName(thread_state_slice->wakeup_tid());
+    std::string process_name = capture_data_->GetThreadName(thread_state_slice->wakeup_pid());
+    tooltip += absl::StrFormat(
+        "<br/>"
```

Comment:
I would prefer an empty line between the description and these lines.

Reply:
I'm still hoping for this empty line... (which I realize is currently not there but I think should be added anyway)

Reply:
Yeah, just added it. I first thought you meant an empty line in the code between `tooltip += ...` and the string above it.

Reply:
Oh whoops, sorry I wasn't clear. Thanks for addressing.